cmis/src/main/java/nl/nn/adapterframework/extensions/cmis/CmisSessionBuilder.java

```diff
@@ -258,7 +261,11 @@ public CloseableCmisSession build(String userName, String password) throws CmisS
 			parameterMap.put(SessionParameter.AUTH_HTTP_BASIC, "true");
 		}
 
-		CloseableCmisSession session = new CloseableCmisSession(parameterMap, null, null, null, null);
+		if(headers != null) {
```

Comment:
Als je een lege map meegeeft ipv `null` dan hoef je hier deze check niet te doen.

Reply:
Iemand anders kan toch deze methode aanroepen met null? Ik zal in de forward-ports de code aanpassen.